src/main/java/duke/Deadline.java

```diff
@@ -0,0 +1,16 @@
+package duke;
+
+public class Deadline extends Task {
+
+    protected String by;
```

Comment:
i think this variable should be private even if it is only going to be accessed within the package! you could add a getter to retrieve this variable